_format/1.1/index.md

```diff
@@ -2153,6 +2153,10 @@ least one of:
     exists; if it doesn't, the client **SHOULD** simply ignore the pointer.
   * `parameter`: a string indicating which URI query parameter caused
     the error.
+  * `header`: a string indicating the name of a single request header which
+    caused the error.
+  If none of these members are provided, the `source` member **SHOULD** be
+  omitted.
```

Comment:
It would be less likely to be overlooked if we move this line up to the definition of the source. In the issue, we also discussed recommending only one member gets set. I think the following is still backward compatible.

```
`source`: an object containing reference to the primary source of the error.
  It **SHOULD** include one of the following members or be omitted:
```

Reply:
@richard-rance: Agreed; incorporated your suggestion.